### ISSUE TITLE
account_file is not defined but got error:  provider.google: "account_file": [REMOVED] Use the credentials field instead

### ISSUE THREAD

**Issue by sirmax123** [hashicorp/terraform#12766]
I'm trying to apply simple configuration after update to terraform 0.9.0 and got the following error:

```
terraform  plan
provider.google.account_file
  Enter a value:

provider.google.credentials
  Enter a value:

1 error(s) occurred:

* provider.google: "account_file": [REMOVED] Use the credentials field instead
```
```
bash-3.2$ terraform  version
Terraform v0.9.0
```
My .tf file is :

```
provider "google" {
    project     = "project"
    region      = "us-central1"
}

resource "google_compute_health_check" "test-health-check" {
    name               = "health-check"
    check_interval_sec = 5
    timeout_sec        = 5

    tcp_health_check {
        port = "8500"
    }
}
```
(and no other files in folder)

For both values during `terraform plan` I use <enter> as answer to variable requests.

This .tf file worked fine before update and I do not have `account_file` defined in .tf

**Comment by danawillow** [comment]
I believe this is fixed in #12729. @paddyforan, anything people should do in the meantime?

**Comment by cblecker** [comment]
I'm experiencing this one too, making 0.9.0 useless for my google resources. :(

**Comment by paddyforan** [comment]
Hi! Sorry for the issues. This is a bug in the GCP provider for Terraform 0.9.0. It only exists in the GCP provider, and only for Terraform 0.9.0. For `terraform import`, the known workaround is to just hit Enter when prompted to the account_file input. For `terraform plan` and `terraform apply`, there is no known workaround at this time.

We will be issuing a 0.9.1 release with a fix soon, and are investigating the underlying problems that led to this issue. Apologies for any trouble this has caused.

**Comment by stack72** [comment]
Closed via #12729 

and released in Terraform 0.9.1
